partner_validation_purchase/models/res_partner.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# © 2018 Akretion
+# © 2018 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models, api, _
+from odoo.exceptions import Warning
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'Warning'](https://app.codacy.com/app/numigi/odoo-partner-addons/pullRequest?prid=2609146)